cupyx/scipy/sparse/_csr.py

```diff
@@ -262,10 +262,10 @@ def __truediv__(self, other):
             # which can use lots of memory.
             self_dense = self.todense().astype(dtype, copy=False)
             return self_dense / other.todense()
-        raise NotImplementedError
+        raise AssertionError
```

Comment:
This one is due to type checking.

```suggestion
        return NotImplemented
```